database/drivers/postgres/postgres.go

```diff
@@ -0,0 +1,39 @@
+package postgres
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	// shut up golint
```

Comment:
Golint just wants to love ❤️

Reply:
While i stand by my original comment i will update it :D